src/Microsoft.PowerShell.Commands.Management/commands/management/Service.cs

```diff
@@ -1506,17 +1609,7 @@ public string Description
         [Parameter]
         [Alias("StartMode", "SM", "ST")]
         [ValidateNotNullOrEmpty]
-        public ServiceStartMode StartupType
-        {
-            get { return startupType; }
-            set
-            {
-                startupType = value;
-            }
-        }
-        // We set the initial value to an invalid value so that we can
-        // distinguish when this is and is not set.
-        internal ServiceStartMode startupType = (ServiceStartMode)(-1);
+        public ServiceStartupType StartupType { get; set; } = (ServiceStartupType)(-1);
```

Comment:
It seems the change is not related to the PR purpose. I'd prefer to move it in separate PR.